### PR TITLE
chore: add DevITJobs to users.md

### PR DESCRIPTION
**Pull request by MariaAbadi**
DevITJobs.com is a job board with a transparent and inclusive approach to job offers (each employer is required to provide the tech stack and salary range). We are using PouchDB for the employer app to enable offline local edits of the job ads which can be synced afterwards.

### CODE REVIEW

**Comment by SourceR85** [comment]
SPAM

**Comment by SourceR85** [comment]
Your profile looks suspicious you didn't explain anything in your commit message,
and your contribution didn't explain anything about how you're related to PouchDB or it's ecosystem.

To me it just looks like an advertisement campaign.

**Comment by SourceR85** [comment]
I'm not talking about your gender, ethnicity, religion, sexuality or anything else of your personality (I treat everyone equally when I talk IT.…)

I refer to your activity in your GH profile [^1], and the commit message[^2] you're missing to explain who the company is, and the lack of information about how DevITJobs is related to PouchDB or it's co-projects (there's not a single word in the link description, of how you and your company is using PouchDB or how it adds value to the ecosystem)

If you just want to have a discussion with our community, please join us on slack.

[^1]: grafik
[^2]: grafik

**Comment by SourceR85** [comment]
> DevITJobs.com is a job board with a transparent and inclusive approach to job offers (each employer is required to provide the tech stack and salary range). We are using PouchDB for the employer app to enable offline local edits of the job ads which can be synced afterwards.

That's useful information 👍
Please update your [commit description](https://github.com/pouchdb/pouchdb/pull/8987#issue-2526143842) with that information and I redirect it to someone to make a final decision.

File: docs/users.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ A list of known products and services that are using PouchDB.
 ## Delta
 [Delta](http://delta.octavore.com/) is a command-line utility for text diffs. View split diffs in the browser with syntax highlighting, or in the command-line using the --cli flag.
 
+## DevITJobs
+[DevITJobs](http://devitjobs.com) is a job board for the tech industry with mandatory salary ranges & tech stacks
```
Comment (SourceR85) [inline docs/users.md:34]:
```suggestion
[DevITJobs](http://devitjobs.com) is a job board for the tech industry with mandatory salary ranges & tech stacks.
```

**Review by SourceR85 — CHANGES_REQUESTED** [review]

**Comment by janl** [comment]
Hi @MariaAbadi, thank you for your contribution and let me apologise on behalf of the project. We did not handle this as we should based on the standards we set out for ourselves.

I want to emphasise that @SourceR85 did not act alone, I also had glanced at this and quickly concluded that it trips up my manual spam filter sensibilities just based on the text. That was clearly wrong and we should have done better.

I understand your frustration with how this went here and with open source in general and I hope by making amends we can at least set a foundation of regaining your interest in contributing PouchDB.

Thank your being a part of this community and for caring enough about it to contribute.

Do let us know if you have any additional questions or comments, either here or in private (my email address is visible on my GitHub profile).

Best
Jan
—

**Review by janl — APPROVED** [review]

**Pull request merged**